expat/lib/xmlparse.c

```diff
@@ -6322,6 +6480,117 @@ normalizePublicId(XML_Char *publicId)
   *p = XML_T('\0');
 }
 
+static LIMIT *
+limitCreate(const XML_Memory_Handling_Suite *ms)
```

Comment:
We may also need something like `limitReset` to be used in `XML_ParserReset`.

Reply:
Good catch! I'll add it to ``XML_ParserReset``.